packages/contracts-bedrock/src/deployment/SystemDictator.sol

```diff
@@ -32,7 +32,6 @@ contract SystemDictator is OwnableUpgradeable {
     struct GlobalConfig {
         AddressManager addressManager;
         ProxyAdmin proxyAdmin;
```

Comment:
so this change is not proxy compatible, but we don't need this contract/change anymore for sepolia right? again on mainnet?

Reply:
No, I don't know why the `ProxyAdmin` is not behind the proxy.

Reply:
no sorry I meant the whole SystemDictator contract changes (the addition of superchainconfig impl slot), but all good since we don't need to update SystemDictator anymore